src/components/icons/icons.test.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import IonIcon from './icons';
+import { IonIconProps } from './icons';
+import { iconType } from './svgs/icons';
+
+const defaultIcon: IonIconProps = {
+  type: 'pencil',
+};
+
+const sut = (props = defaultIcon) => render(<IonIcon {...props} />);
+const getIcon = (type: iconType) => screen.getByTestId(`ion-icon-${type}`);
+
+describe('IonIcon', () => {
+  it('should render default icon of type pencil', () => {
+    sut();
+    const icon = getIcon('pencil');
+    expect(icon).toBeTruthy();
+    expect(icon.getAttribute('width')).toBe('24');
+    expect(icon.getAttribute('height')).toBe('24');
+    expect(icon.getAttribute('fill')).toBe('#282B33');
+  });
```

Comment:
```suggestion
  it('should render default size icon of type pencil', () => {
    sut();
    const icon = getIcon('pencil');
    expect(icon).toBeTruthy();
    expect(icon.getAttribute('width')).toBe('24');
    expect(icon.getAttribute('height')).toBe('24');
  });
```

and create a new spec to test this
    expect(icon.getAttribute('fill')).toBe('#282B33');